src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -124,6 +127,9 @@
     this.repairUnit = repairUnit;
     this.repairRunner = repairRunner;
     this.segmentFailed = new AtomicBoolean(false);
+    this.gotCompleteNotification = new AtomicBoolean(false);
+    this.gotFailedNotification = new AtomicBoolean(false);
+    this.gotSuccessNotification = new AtomicBoolean(false);
```

Comment:
can we use the following shorter names for these plesae?
{{completeNotified}}, {{failedNotified}}, {{successNotified}}